class/book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
+end
```

Comment:
- [ ] Great job so far creating the associations between your classes. However, In your method for adding rentals please make sure to pass the correct `parameters`. The book itself should be sent as a `parameter` to create the new rental achieving the `has-many` association.  Instead of passing the rental, you should pass `(person, date)`. Your code should look like below.

```
 def add_rental(person, date)
   Rental.new(date, self, person)
  end
```